readalongs/align.py

```diff
@@ -950,41 +960,44 @@ def save_readalong(
         save_xml(tokenized_xhtml_path, align_results["tokenized"])
 
     audio_path = save_audio(
-        audiofile=audiofile, output_base=output_base, audiosegment=audiosegment
+        audiofile=audiofile, output_base=bundle_base, audiosegment=audiosegment
     )
 
     if "html" in output_formats:
-        html_out_path = output_base + ".html"
+        offline_html_dir = os.path.join(output_dir, "Offline-HTML")
+        html_out_path = os.path.join(offline_html_dir, output_basename + ".html")
```

Comment:
I think the Offline file name should be index.html. `Offile-HTML\index.html`

Reply:
The title is harder to set in the CLI, and will frequently just have the default value, so I'd rather use the `output_basename`. Unless we check that it's not the default title, but let's make a separate issue about this, it's work I don't want to roll into this PR.

Reply:
Added #234 for this.